### ISSUE TITLE
Make Gun course possible with jetboard

### ISSUE THREAD

**Issue by Zedb0T** [OpenGOAL-Mods/OG-NoOOJ-II#7]

**Comment by dallmeyer** [comment]
scatter gun course is possible with circle attack

**Comment by Zedb0T** [comment]
I still think we should consider not having the circle attack as well personally but am open just having it always on if everyone thinks so.

**Comment by dallmeyer** [comment]
yeah im not opposed to that, and either way there might still be more things we can do to make gun course enjoyable

**Comment by Zedb0T** [comment]
You can just run into the dudes and its funny, we might need to adjust the required score

**Comment by Zedb0T** [comment]
Need to make sure the tutorials dont softlock - test in game and not mission select
